stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class MakeProcurementOrderpoint(models.TransientModel):
+    _name = 'make.procurement.orderpoint'
+    _description = 'Make Procurements from Orderpoints'
+
+    item_ids = fields.One2many(
+        'make.procurement.orderpoint.item',
+        'wiz_id', string='Items')
+
+    @api.model
+    def _prepare_item(self, orderpoint):
+
+        if orderpoint.procure_uom_id:
+            product_uom = orderpoint.procure_uom_id
+        else:
+            product_uom = orderpoint.product_uom
+
+        return {
+            'qty': orderpoint.procure_recommended_qty,
+            'uom_id': product_uom.id,
+            'date_planned': orderpoint.procure_recommended_date,
+            'orderpoint_id': orderpoint.id,
+            'product_id': orderpoint.product_id.id,
+            'warehouse_id': orderpoint.warehouse_id.id,
+            'location_id': orderpoint.location_id.id
+        }
+
+    @api.model
+    def default_get(self, fields):
+        res = super(MakeProcurementOrderpoint, self).default_get(
+            fields)
+        orderpoint_obj = self.env['stock.warehouse.orderpoint']
+        orderpoint_ids = self.env.context['active_ids'] or []
+        active_model = self.env.context['active_model']
+
+        if not orderpoint_ids:
+            return res
+        assert active_model == 'stock.warehouse.orderpoint', \
+            'Bad context propagation'
+
+        items = []
+        for line in orderpoint_obj.browse(orderpoint_ids):
+            items.append([0, 0, self._prepare_item(line)])
+        res['item_ids'] = items
+        return res
+
+    @api.multi
+    def make_procurement(self):
+        self.ensure_one()
+        res = []
+        for item in self.item_ids:
+            data = item._prepare_procurement()
+            procurement = self.env['procurement.order'].create(data)
+            procurement.run()
```

Comment:
@lreficent Procurements are run on creation. This line avoid to have the 'procurement_autorun_defer' context option to be used. Is this the wanted behaviour?

Reply:
good point, removing it.